pkg/istio-agent/xds_proxy.go

```diff
@@ -143,8 +143,6 @@ func (p *XdsProxy) StreamAggregatedResources(downstream discovery.AggregatedDisc
 	// Handle downstream xds
 	firstNDSSent := false
 	go func() {
-		defer close(requestsChan) // Indicates downstream close.
```

Comment:
This needs to be handled properly before merge. May be in this PR, or another one.